rubrics/impl/src/java/org/sakaiproject/rubrics/logic/RubricsServiceImpl.java

```diff
@@ -0,0 +1,797 @@
+     * Returns the JSON string for the rubric evaluation
+     * @param toolId the tool id, something like "sakai.assignment"
+     * @param associatedToolItemId the id of the tool item which has a rubric associated to it (e.g. assignment ID)
+     * @param evaluatedItemId  the id of the tool item which is being evaluated using a rubric (e.g. assignment submission ID)
+     * @return
+     */
+    public String getRubricEvaluation(String toolId, String associatedToolItemId, String evaluatedItemId) throws IOException {
+
+        try{
+            URL url = new URL(serverConfigurationService.getServerUrl() + RBCS_SERVICE_URL_PREFIX + "evaluations/search/by-tool-item-and-associated-item-and-evaluated-item-ids?toolId="+toolId+"&itemId="+associatedToolItemId+"&evaluatedItemId="+evaluatedItemId);
+
+            String cookie = "JSESSIONID=" + sessionManager.getCurrentSession().getId() + "." + System.getProperty(SERVER_ID_PROPERTY);
+            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+            conn.setRequestMethod("GET");
+            conn.setRequestProperty("Accept", "application/json");
+            conn.setRequestProperty("Cookie", cookie );
+            conn.setRequestProperty("Authorization", "Bearer" + generateJsonWebToken(toolId));
+
+            if (conn.getResponseCode() != 200) {
+                throw new RuntimeException("Failed : HTTP error code : "
+                        + conn.getResponseCode());
+            }
+
+            BufferedReader br = new BufferedReader(new InputStreamReader(
+                    (conn.getInputStream())));
+
+            String output;
+            StringWriter result = new StringWriter();
+            while ((output = br.readLine()) != null) {
+                result.append(output + "\n");
+            }
+            conn.disconnect();
+            return result.toString();
+
+        } catch (MalformedURLException e) {
+
+            log.warn("Error getting a rubric evaluation " + e.getMessage());
+            return null;
+
+        } catch (IOException e) {
+
+            log.warn("Error getting a rubric evaluation" + e.getMessage());
+            return null;
+        }
+    }
+
+
+    public String generateLang(){
+
+        StringBuilder lines = new StringBuilder();
+        lines.append("var rubricsLang = {");
+
+        Locale locale = rb.getLocale();
+        Set properties = rb.keySet();
+        lines.append("'" + locale.toLanguageTag() + "': {");
+        Iterator keys = properties.iterator();
+        while (keys.hasNext()){
+            String key = keys.next().toString();
+            if (keys.hasNext()) {
+                lines.append("'" + key + "': '" + rb.getString(key) + "',");
+            }else{
+                lines.append("'" + key + "': '" + rb.getString(key) + "'");
+            }
+        }
+
+        lines.append("}");
+        lines.append("}");
+
+        log.debug(lines.toString());
+
+        return lines.toString();
+    }
+
+
+    @Getter
```

Comment:
be nice if these declarations were all moved to the top as that is the pattern used everywhere else.